src/sentry/integrations/discord/webhooks/message_component.py

```diff
@@ -65,6 +75,12 @@ def handle(self) -> Response:
             return self.send_message(NO_IDENTITY)
         self.user = self.request.user
 
+        if not self.group_id:
+            return self.send_message(INVALID_GROUP_ID)
+
+        if not self.group:
+            return self.send_message(INVALID_GROUP_ID)
```

Comment:
```suggestion
        if not self.group_id or not self.group:
            return self.send_message(INVALID_GROUP_ID)
```

you could combine these conditions 🏌️